auth.go

```diff
@@ -90,7 +94,7 @@ func (v *Verifier) VerifyAuthResponse(ctx context.Context, response protocol.Aut
 		}
 
 		// prepare query from request
-		queryBytes, err := json.Marshal(proofRequest.Rules["query"])
+		queryBytes, err := json.Marshal(proofRequest.Query["query"])
```

Comment:
that is not right. proofRequest.Query is already a query

Reply:
also, why do we have AuthV2Request and not auth v2 response? WHERE IS CONSISTENCY?
also, I think we don't need V2 at all. even on the protocol level. and operate with v2 here.(e.g. createV2request)

Reply:
`proofRequest.Query` is correct. Check iden3comm.

Reply:
Done